### DISCUSSION TITLE
历史数据的展示时间跨度可以调整吗？

### DISCUSSION THREAD

**Discussion by lixdx** [Mirouterui/mirouter-ui#35] [Q&A] [answered]
历史数据的展示时间跨度可以调整吗？想改为30天或者15天就够。或者有个选择的项目。
<img width="348" alt="image" src="https://github.com/user-attachments/assets/a5d7b983-2de8-49d8-96a6-8c292e8599e5">

**Comment by thun888** [comment]
下次更新加上

**Reply by lixdx** [reply]
哈哈，个人建议
谢谢大佬

**Comment by thun888** [comment, marked as answer]
当然，你也可以通过滑动图表下方的滑动条来控制显示范围

**Reply by lixdx** [reply]
每个graph需要单独调整，不太方便哈哈哈

**Reply by thun888** [reply]
除了流量信息外，图表数据是同步调整的
